help/faq/texprobs.md

```diff
@@ -0,0 +1,88 @@
+# Why doesn't my processed TeX submission look the way I expected it?
+
+There are a number of common reasons why the processed version of TeX submissions to arXiv may not appear the same as it does when processed locally. In most cases there are fixes that can be applied and the submission replaced to correct the problems.
+
+*   [Lines in the table of contents don't wrap](#toc_links)
+*   [Fuzzy fonts in PDF](#fuzzy_pdf)
+*   [PostScript figures lose quality in the arXiv-generated PDF](#distiller_params)
+*   [Extra white space around figures covers up some of the text](#powerpoint)
+*   [Why don't the labels for my figures appear when I use Feynmf?](feynmf#nolabel)
+*   [Why are the margins different? Why is my text truncated?](dvips)
+*   [Why does my paper give the wrong date?](today)
+*   [Why do my citations appear in long form \[1,2,3,4\] instead of short form \[1-4\]?](citelinks)
```

Comment:
Broken link, found at faq/citelinks

Reply:
curious because this page exists
https://github.com/arXiv/arxiv-docs/blob/develop/help/faq/citelinks.md

@eawoods 
Is it the special characters in the linked text being rendered that is messing it up?